sound/soc/intel/boards/cml_rt1011_rt5682.c

```diff
@@ -488,6 +494,14 @@ static struct snd_soc_codec_conf rt1011_conf[] = {
 		.dlc = COMP_CODEC_CONF("i2c-10EC1011:01"),
 		.name_prefix = "WR",
 	},
+	{
```

Comment:
I would add a comment that we have a single configuration structure for the two 2 and 4 channels, it's useful for reviewers.